packages/cactus-plugin-ledger-connector-corda/src/main/typescript/web-services/get-flow-endpoint-v1.ts

```diff
@@ -0,0 +1,112 @@
+import { Express, Request, Response } from "express";
+
+import {
+  IWebServiceEndpoint,
+  IExpressRequestHandler,
+  IEndpointAuthzOptions,
+} from "@hyperledger/cactus-core-api";
+
+import {
+  registerWebServiceEndpoint,
+  handleRestEndpointException,
+} from "@hyperledger/cactus-core";
+
+import {
+  Checks,
+  IAsyncProvider,
+  Logger,
+  LoggerProvider,
+  LogLevelDesc,
+} from "@hyperledger/cactus-common";
+
+import OAS from "../../json/openapi.json";
+import { PluginLedgerConnectorCorda } from "../plugin-ledger-connector-corda";
+
+export interface IFlowStatusResponseEndpointV1Options {
+  logLevel?: LogLevelDesc;
+  apiUrl?: string;
+  holdingIDShortHash: string;
+  clientRequestID: string;
+  connector: PluginLedgerConnectorCorda;
+}
+
+export class FlowStatusResponseEndpointV1 implements IWebServiceEndpoint {
+  public static readonly CLASS_NAME = "FlowStatusResponseEndpointV1";
+
+  private readonly log: Logger;
+  private readonly apiUrl?: string;
```

Comment:
@adrianbatuto Please make this a mandatory field and then ensure in the constructor that an error is thrown if it's a blank string or otherwise falsy. This applies to all the endpoint implementation classes not just the one I'm commenting on (but I don't want to copy-paste-spam the review comments)